.sync.yml

```diff
@@ -1,3 +1,5 @@
 ---
```

Comment:
Since you changed the CI workflow, shouldn't that be unmanaged too now?

Reply:
I'm still experimenting with it. I might add a patch to gha-puppet to support custom package installation during CI.